orc8r/cloud/go/services/bootstrapper/servicers/registration/cloud_registration.go

```diff
@@ -0,0 +1,154 @@
+package registration
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"time"
+
+	"magma/orc8r/lib/go/protos"
+
+	"github.com/golang/glog"
+	"github.com/golang/protobuf/ptypes/timestamp"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+const (
+	rootCAFilePath = "/var/opt/magma/certs/rootCA.pem"
+
+	// length of timeout for the nonce
+	timeoutDuration = 30 * time.Minute
+
+	// number of characters that the nonce will have
+	// 52^7 chance = 1.0280717e+12
+	nonceLength = 7
```

Comment:
Let's make this 30 actually -- I wanna get us to at least 150 bits of entropy, since this will be the only form of auth required to be presented

- https://www.omnicalculator.com/other/password-entropy
- https://iocane.com.au/talking-passwords-and-entropy/

Another option I thought could be fun + good for UX -- could use diceware-generated tokens

- https://github.com/sethvargo/go-diceware
- Though this library would then require a 12-word password to generate the same entropy as the 30-char random string. Up to you 🙂

Reply:
I thought we wanted it to be typeable? and easy to copy somewhere else?